.github/ISSUE_TEMPLATE/add-a-new-evm-network.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Add a new EVM network
+about: I want to begin the process to add a new evm network to hmt-escrow
+title: ''
+labels: feature
+assignees: posix4e
+
+---
+
+HMT-Escrow supports arbitrary EVM networks but there's a bit of work to do to make a network first class and make sure that it works. All of our EVM network launches are supported by gitcoin but you can gain a yearly bonus by being our network sponsor. 
+
+### Steps to receive gitcoin bounty
+- [ ] Launch the contracts on the devnet and make sure that everything is working
```

Comment:
This also involves a bridge setup between networks. In case this new network has test networks, there should be a bridge between test networks(e.g. Ethereum Rinkeby and Polygon Mumbai)

Reply:
Yep